File: SharedApi/Models/Unavailability.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SharedApi.Models
{
	public partial class Unavailability
	{
		public int ID { get; set; }
		public int ProjectID { get; set; }
		public string Reason { get; set; }
		public string Description { get; set; }
		public System.DateTime From { get; set; }
		public System.DateTime To { get; set; }
	}
}

File: MobileApp/zoliky_teachers/lib/utils/api/models/Unavailability.dart
import 'package:zoliky_teachers/utils/interfaces/IDbObject.dart';

class Unavailability extends IDbObject {
  int id;
  int projectID;
  String reason;
  String description;
  DateTime from;
  DateTime to;

  Unavailability();

  Unavailability.fromJson(Map<String, dynamic> json)
      : id = json['ID'],
        projectID = json['ProjectID'],
        reason = json['Reason'],
        description = json['Description'],
        from = DateTime.parse(json['From']),
        to = DateTime.parse(json['To']);
}
